uint8 floor
uint8 room
uint8 bed
uint8 status
